packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -57,15 +59,16 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
 
       this.webSocketManager = new WebSocketManager({webex: this.$webex});
 
-      this.connectionService = new ConnectionService(
-        this.webSocketManager,
-        this.getConnectionConfig()
-      );
+      this.connectionService = new ConnectionService({
```

Comment:
Sticking to the old way of doing it, but with an options.
We need subscribeRequestConfig so that we can re-initialise the websocket connection rather than once again calling the cc.connectWebSocket method here, we directly register it and connect (same as what agent desktop does).

Reply:
is there a possibility to combine them and move it to services layer completely.